internal/pkg/utils/etcdlogger/etcdlogger.go

```diff
@@ -169,6 +169,15 @@ func (v *kvWrapper) start(op *etcd.Op, opName, key, value string) string {
 				out.WriteString(strconv.FormatInt(op.Rev(), 10))
 			}
 		}
+		if op.IsCountOnly() {
+			out.WriteString(" | count only")
+		}
+		if op.IsKeysOnly() {
+			out.WriteString(" | keys only")
+		}
+		if op.IsSerializable() {
+			out.WriteString(" | serializable")
+		}
```

Comment:
etcd query can have these options set, I add them to logs, you can find the meaning of individual options in the etcd documentation.